gulpfile.js

```diff
@@ -20,7 +22,9 @@ require('./').gulpDoxx(require('gulp'),doxxOptions);
 
 
 var ghPagesOptions = {
-
+  config:{
+    remoteUrl: 'git@github.com:pattern-library/pattern-library-utilities.git'
```

Comment:
@scottnath I think this only allows publishing to the gh-pages for the Pattern Library account and only for those who have write access. That by itself isn't really a problem. However, it seems that you wish to either forbid, or else make it difficult, for developers to publish a copy of this repo to their own gh-pages?